firestore-translate-text/PREINSTALL.md

```diff
@@ -9,6 +9,22 @@ You specify the desired target languages using ISO-639-1 codes. You can find a l
 
 If the original non-translated field of the document is updated, then the translations will be automatically updated, as well.
 
+#### Multiple collections for translations
+
+To translate through `multiple collections`, it is recommended to install multiple versions of the extension but with a alternative `collection path` for each installation.
+
+There is currently no limit on how many times the extensions can be reinstalled.
+
+#### Multiple field translations
+
+Similar to multiple collections, to translate with `multiple fields` it is recommended to install multiple versions of the extension but with an alternative `input field` for each installation.
```

Comment:
multiple translations happen on the same extension instance. Rather than pass a string to the input field for translation, you can pass an object and every property will be translated. Please update.